doc/Command-Reference.md

```diff
@@ -10397,6 +10397,38 @@ This command removes all unused images from the device, leaving only the current
   No image(s) to remove
   ```
 
+**sonic-installer get-fips**
+
+This command is used to get the fips status.
+
+- Usage:
+  ```
+  sonic-installer get-fips [<image_name>]
+  ```
+
```

Comment:
```suggestion
If the image is not specified, the next boot image will be used.

```
Add a note.

Reply:
@mdanish-kh , thanks for your suggestion.